.buildkite/kind-in-docker.yml

```diff
@@ -0,0 +1,17 @@
+- label: kind in Docker
+  instance_size: large
+  image: golang:1.17
+  commands:
+    # Install Go
+    - export PATH=$PATH:/usr/local/go/bin
+
+    # Install kind
```

Comment:
Does this mean that we always need to install Kind and Docker each time we trigger the CI pipeline?

Reply:
Yes, that's what's happening currently.  Any suggestions for how to optimize it in the future?

Reply:
I guess we could build the dependencies into a Docker image and upload the image to some repository?

Reply:
We could consider creating a dedicated image with Kind and Docker for testing purposes. Would you mind opening an issue for this?

Reply:
Sure, done https://github.com/ray-project/kuberay/issues/1247